controllers/rabbitmqcluster_controller_test.go

```diff
@@ -957,6 +958,28 @@ var _ = Describe("RabbitmqClusterController", func() {
 				return c.Image
 			}, 3).Should(Equal("my-great-image-2"))
 		})
+
+		It("can reset a controller-set field to default", func() {
```

Comment:
Should we make the test description more specific here? Our controller is only treating pod and container SecurityContext as special cases, and it currently does not handle any other fields